### ISSUE TITLE
Filename attachment

### ISSUE THREAD

**Issue by thevidocq** [xme/dockers#11]
Hi, is possible upload attachment without the temporary filename rename? For example payload security analysis fails because the extension is wrong. Ex: /tmp/randormdir/attachmnetname.doc Thanks!

**Comment by xme** [comment]
Good remark! I'll work on this.

**Comment by thevidocq** [comment]
Many thanks!

**Comment by xme** [comment]
I just did a commit to solve this issue (1fe86af). Let me know!

**Comment by xme** [comment]
Yes I could but IMHO there is a (small) risk of conflict with common emails if you receive a lot of them at the same time.
